test/sanity/requirements.txt

```diff
@@ -2,3 +2,6 @@ packaging  # needed for update-bundled and changelog
 sphinx ; python_version >= '3.5' # docs build requires python 3+
 sphinx-notfound-page ; python_version >= '3.5' # docs build requires python 3+
 straight.plugin ; python_version >= '3.5' # needed for hacking/build-ansible.py which will host changelog generation and requires python 3+
+
+# changelog build requires python 3.6+
+antsibull ; python_version >= '3.6'
```

Comment:
We don't yet have a way to limit code-smell tests to specific python versions. We'll need to add that so the test can be skipped on Python 3.5.

Reply:
I've implemented `minimum_python_version` in 110e542784b5ceb41cd9bf33964cef06859686c9